### PR TITLE
Release v0.15

### PR DESCRIPTION
**Pull request by ludwigschwardt**
This updates the changelog, which will get merged back into master. The final commit will be tagged as v0.15 and released.

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.15 (2020-03-13)
+-----------------
+* Improve S3 chunk store: check tokens, improve timeouts and retries (#272 - #277)
+* Apply flux calibration if available (#278, #279)
+* Improve mvf_rechunk and mvf_read_benchmark scripts (#280, #281, #284)
+* Fix selection by target description (#271)
```
Comment (bmerry) [inline NEWS.rst]: Maybe add a note that Python 2 support is deprecated?

File: NEWS.rst [hunk 2]
```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.15 (2020-03-13)
+-----------------
+* Improve S3 chunk store: check tokens, improve timeouts and retries (#272 - #277)
```
Comment (bmerry) [inline NEWS.rst]: So you want to say anything specific about truncated reads?

File: NEWS.rst (hunk 1 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Ah yes, missed that since I only looked at PR merge commits...

File: NEWS.rst (hunk 2 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Good idea.

**Pull request merged**